src/main/resources/schemas/nhinc/common/AuditLog.xsd

```diff
@@ -1,175 +1,188 @@
+    <xsd:complexType name="DocRetrieveResponseMessageType">
+        <xsd:sequence>
+            <xsd:element ref="ihe:RetrieveDocumentSetResponse"/>
+            <xsd:element name="assertion" type="nccommon:AssertionType"/>
+        </xsd:sequence>
+    </xsd:complexType>
+    <xsd:complexType name="LogDocRetrieveResultRequestType">
+        <xsd:sequence>
+            <xsd:element name="message" type="auditlog:DocRetrieveResponseMessageType"/>
+            <xsd:element ref="auditlog:direction"/>
+            <xsd:element ref="auditlog:interface"/>
+        </xsd:sequence>
+    </xsd:complexType>
+    <xsd:element name="LogDocRetrieveResultRequest" type="auditlog:LogDocRetrieveResultRequestType"/>
+    <!-- begin doc query response -->
+    <!-- begin find audit events request -->
+    <xsd:complexType name="FindAuditEventsMessageType">
+        <xsd:sequence>
+            <xsd:element ref="auditmessage:FindAuditEvents"/>
+            <xsd:element name="assertion" type="nccommon:AssertionType"/>
+        </xsd:sequence>
+    </xsd:complexType>
+    <xsd:complexType name="LogFindAuditEventsRequestType">
+        <xsd:sequence>
+            <xsd:element name="message" type="auditlog:FindAuditEventsMessageType"/>
+            <xsd:element ref="auditlog:direction"/>
+            <xsd:element ref="auditlog:interface"/>
+        </xsd:sequence>
+    </xsd:complexType>
+    <xsd:element name="LogFindAuditEventsRequest" type="auditlog:LogFindAuditEventsRequestType"/>
+    <!-- end find audit events request -->
+    <!-- begin find audit events response -->
+    <xsd:complexType name="FindAuditEventsResponseMessageType">
+        <xsd:sequence>
+            <xsd:element ref="auditmessage:FindAuditEventsResponse"/>
+            <xsd:element name="assertion" type="nccommon:AssertionType"/>
+        </xsd:sequence>
+    </xsd:complexType>
+    <xsd:complexType name="LogFindAuditEventsResultRequestType">
+        <xsd:sequence>
+            <xsd:element name="message" type="auditlog:FindAuditEventsResponseMessageType"/>
+            <xsd:element ref="auditlog:direction"/>
+            <xsd:element ref="auditlog:interface"/>
+        </xsd:sequence>
+    </xsd:complexType>
+    <xsd:element name="LogFindAuditEventsResultRequest" type="auditlog:LogFindAuditEventsResultRequestType"/>
+    <!-- end find audit events response -->
+    <!-- begin generic log message -->
+    <xsd:complexType name="LogEventRequestType">
+        <xsd:sequence>
+            <xsd:element ref="auditmessage:AuditMessage"/>
+            <xsd:element ref="auditlog:direction"/>
+            <xsd:element name="assertion" type="nccommon:AssertionType"/>
+            <xsd:element name="EventTimestamp" minOccurs="0" type="xsd:dateTime"/>
+            <xsd:element name="EventID" minOccurs="0" type="xsd:string"/>
+            <xsd:element name="EventType" minOccurs="0" type="xsd:string"/>
+            <xsd:element name="EventOutcomeIndicator" minOccurs="0" type="xsd:integer"/>
```

Comment:
The above four Event* elements are NOT NULL in the database definition; should this be reflected here?